front/lib/api/auth0.ts

```diff
@@ -15,3 +24,80 @@ export function getAuth0ManagemementClient(): ManagementClient {
 
   return auth0ManagemementClient;
 }
+
+/**
+ * Verify an Auth0 token.
+ * Not meant to be exported, use `getUserFromAuth0Token` instead.
+ */
+export function verifyAuth0Token(idToken: string): Promise<jwt.JwtPayload> {
+  const client = jwksClient({
+    jwksUri: AUTH0_VERIFY_TOKEN,
+    cache: true,
+    rateLimit: true,
+  });
+
+  function getKey(header: jwt.JwtHeader, callback: jwt.SigningKeyCallback) {
```

Comment:
`nit`
Let's avoid nested function within function to ease visual parsing. You can use anonymous functions instead.